packages/trader/src/AppV2/Components/MarketCategoryItem/market-category-item.tsx

```diff
@@ -30,15 +31,28 @@ const MarketCategoryItem = forwardRef(
             setIsFavorite(favoriteSymbols.includes(item.symbol));
         }, [favoriteSymbols, item.symbol]);
 
-        const handleSelect = async (e: React.MouseEvent<HTMLSpanElement>) => {
-            const symbol = (e.target as HTMLSpanElement).getAttribute('data-symbol');
-            setSelectedSymbol(symbol ?? '');
+        const clickAndKeyEventHandler = (
```

Comment:
But why you are creating clickAndKeyEventHandler  if you've already imported it?

Reply:
![Uploading Screenshot 2024-08-06 at 12.14.12 PM.png…]()

Reply:
![Screenshot 2024-08-06 at 12 14 12 PM](https://github.com/user-attachments/assets/d7e8aa97-e00b-4e48-8b78-c05429361c19)

Reply:
5pm after meeting brain things